### PR TITLE
Add cesium and leaflet examples

### PR DESCRIPTION
**Pull request by loicgasser**

### CODE REVIEW

File: chsdi/static/doc/source/api/examples.rst [hunk 1]
```diff
@@ -88,9 +103,21 @@ Communication between Iframe-embeded map and parent window
...
+      codepen: 'zBEYGE',
+      cat: 4
+    }, {
+      label: 'WMTS in EPSG:3857 using Leaflet',
+      codepen: 'grGOLV',
+      cat: 5
+    }, {
+      label: 'WMS in EPSG:3857 using Leaflet',
+      codepen: 'JKAjWk',
+      cat: 5
```
Comment (gjn) [inline chsdi/static/doc/source/api/examples.rst]: We could put wms/wmts in the same example, no?

Comment (loicgasser) [inline chsdi/static/doc/source/api/examples.rst]: We could, do you want me to?

**Comment by gjn** [comment]
Yes, I would. Then we can link diretly to the example, instead of the collection.

**Comment by loicgasser** [comment]
There are currently no category that fits these new examples (namely JS API example, OpenLayers 3 examples, GeoAdmin iframe examples).

**Comment by oterral** [comment]
Nice new examples

**Pull request merged**